README.md

```diff
@@ -110,43 +65,33 @@ sudo rm -rf **\~/.solcx**
 sudo rm -rf **\~/.vvm**
 ```
 
-## Environment Variables:
+## Playing with Ape
 
-Some plugins require environment variables to connect to their external systems, such project IDs or API keys.
-Follow instructions from individual plugin documentations, such as:
+After you installed Ape, you can run `ape --version` to make sure it works and it is the latest version.
 
-- [ape-alchemy](https://github.com/ApeWorX/ape-alchemy/blob/main/README.md#quick-usage)
-- [ape-infura](https://github.com/ApeWorX/ape-infura#readme)
+There are two ways to interact with Ape:
 
-Generally, set environment variables by doing the following:
+- [CLI Reference](https://docs.apeworx.io/ape/latest/index.html)
 
-```bash
-# Used by the `ape-infura` plugin
-export WEB3_INFURA_PROJECT_ID=<YOUR_INFURA_PROJECT_ID>
-# Used by the `ape-alchemy` plugin
-export WEB3_ALCHEMY_API_KEY=<YOUR_ALCHEMY_KEY>
-```
+- [Python Reference](https://docs.apeworx.io/ape/latest/index.html)
 
-Place these in environment files, such as your `.bashrc` or `.zshrc`.
+Ape is both a CLI tool and a Python SDK.
 
-## Quick Usage
+The CLI tool contains all the Ape commands and the Python SDK contains the classes and types needed to compose scripts, console actions, and tests.
 
-Use `-h` to list all the commands:
+## **Ape Modular Plugin System:**
 
-```bash
-ape -h
-```
+Our [list of plugins](https://www.apeworx.io/) is the way we want you to have the most interoperable experience with the Web3 Ecosystem.
```

Comment:
"ecosystem" is kind of an overloaded word in our framework, best to avoid here.

```suggestion
Our [list of plugins](https://www.apeworx.io/) is the best way to have the most interoperable experience with Web3.
```